homeassistant/components/arcam_fmj/const.py

```diff
@@ -5,6 +5,8 @@
 SIGNAL_CLIENT_STOPPED = "arcam.client_stopped"
 SIGNAL_CLIENT_DATA = "arcam.client_data"
 
+EVENT_TURN_ON = "arcam.turn_on"
```

Comment:
The prefix should be the integration domain as standard.
```suggestion
EVENT_TURN_ON = "arcam_fmj.turn_on"
```